test/User-test.js

```diff
@@ -190,10 +190,32 @@ describe('User', function() {
     const newIngAtIndex2 = user.pantry[2].ingredient;
     const updatedPantryLength = user.pantry.length;
 
-    expect(originalFlourAmount).to.equal(5);
+    expect(originalFlourAmount).to.equal(9);
```

Comment:
Same case here. Removing the negative sign above and here (lines 169 and 182, respectively) should ensure that the origina expect statements work.